### ISSUE TITLE
Support for disable generating response headers

### ISSUE THREAD

**Issue by doggy8088** [kevinswiber/postman2openapi#73]
The response headers in Postman are mostly trivial. Is it possible add an option to disable generating these headers in OpenAPI Specification output?

**Comment by kevinswiber** [comment]
Yes!  Looking at adding configuration options after a fairly significant refactor that's in the works.
